StepsToCare/CareEntry.ascx.cs

```diff
@@ -733,6 +818,159 @@ protected void bddlAddWorker_SelectionChanged( object sender, EventArgs e )
             bddlAddWorker.ClearSelection();
         }
 
+        protected void dvpCategory_SelectedIndexChanged( object sender, EventArgs e )
+        {
+            PreviewAssignedPeople( null );
+            dtbDetailsText.Focus();
+        }
+
+        protected void gAssignedPersons_RowDataBound( object sender, GridViewRowEventArgs e )
+        {
+            var phCountOrRole = e.Row.ControlsOfTypeRecursive<PlaceHolder>().FirstOrDefault();
+            var assignedPerson = e.Row.DataItem as AssignedPerson;
+
+            if ( phCountOrRole == null || assignedPerson == null )
+            {
+                return;
+            }
+            var returnStr = assignedPerson.Type.ToString();
+            if ( assignedPerson.TypeQualifier.IsNotNullOrWhiteSpace() )
+            {
+                var typeQualifierArray = assignedPerson.TypeQualifier.Split( '^' );
+                if ( assignedPerson.Type == AssignedType.Worker )
+                {
+                    //Format is [0]Count^[1]HasAgeRange^[2]HasCampus^[3]HasCategory^[4]HasGender
+                    returnStr = $"Worker ({typeQualifierArray[0]})";
+                }
+                else if ( assignedPerson.Type == AssignedType.GroupRole && typeQualifierArray.Length > 2 )
+                {
+                    //Format is [0]GroupRoleId^[1]GroupTypeId^[2]Group Type > Group Role
+                    returnStr = typeQualifierArray[2];
+                }
+            }
+            phCountOrRole.Controls.Add( new LiteralControl( returnStr ) );
+
+        }
+
+        protected void cbCustomFollowUp_CheckedChanged( object sender, EventArgs e )
+        {
+            pnlRecurrenceOptions.Visible = cbCustomFollowUp.Checked;
+            BindAssignedPersonsGrid();
+        }
+
+        protected void dvpStatus_SelectedIndexChanged( object sender, EventArgs e )
```

Comment:
Is there a reason status affects the preview?

Reply:
Only thing I was able to determine was I did it intentionally for if you had any attributes with a qualifier status, such as if you had particular attributes set to "Long Term Care".